### PR TITLE
(feat): add openebs pool pod failure experiment CR

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  add openebs pool pod delete experiment CR

### CODE REVIEW

**Review by chandankumar4 — DISMISSED** [review]

**Review by chandankumar4 — APPROVED** [review]

File: charts/openebs/openebs-pool-pod-failure/experiment.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      value: '' 
+
+    - name: CHAOS_ITERATIONS
+      value: '2'
+
+    - name: DATA_PERSISTENCE
+      value: ''
+
+    labels:
+      name: openebs-pool-failure
```
Comment (chandankumar4) [inline charts/openebs/openebs-pool-pod-failure/experiment.yaml]: It should be `openebs-pool-pod-failure`

File: charts/openebs/openebs-pool-pod-failure/experiment.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    configmaps:
+    - name: pool-pod-delete
```
Comment (chandankumar4) [inline charts/openebs/openebs-pool-pod-failure/experiment.yaml]: Why are we using the name `delete` here?

**Review by chandankumar4 — CHANGES_REQUESTED** [review]
Have some small comment, PTAL

File: charts/openebs/openebs-pool-pod-failure/experiment.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        - ""
+        - "extensions"
+        - "apps"
+        - "batch"
+        - "litmuschaos.io"
+        - "openebs.io"
+      resources:
+        - "daemonsets"
+        - "deployments"
+        - "replicasets"
```
Comment (ksatchit) [inline charts/openebs/openebs-pool-pod-failure/experiment.yaml:26]: sts

File: charts/openebs/openebs-pool-pod-failure/openebs-pool-failure.chartserviceversion.yaml [hunk 4]
```diff
@@ -1,29 +1,32 @@
...
-  displayName: openebs-pool-failure 
+  displayName: openebs-pool-pod-failure
   categoryDescription: >
      Kill the pool pod and check if gets scheduled again. 
   keywords:
     - Kubernetes
     - Storage Pool
     - OpenEBS
+  Platforms:
+    - Gke
```
Comment (ksatchit) [inline charts/openebs/openebs-pool-pod-failure/openebs-pool-failure.chartserviceversion.yaml]: GKE

**Review by chandankumar4 — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]
/lgtm

**Review by ksatchit — APPROVED** [review]

**Pull request merged**